bin/smoke-test.sh

```diff
@@ -58,7 +58,7 @@ extended=0
 tokenType=macaroon
 workarounds=""
 debugOutput=0
-vo=dteam
+vo=${SMOKE_TEST_VO:-dteam}
```

Comment:
This is OK, but I'm wondering if it's necessary.

It is (I believe) redundant, as the VO may be specified by the `-v` command-line option.

The `smoke-em.sh` script doesn't seem to set this environment variable, so I'm guessing the intention is that the `SMOKE_TEST_VO` environment variable is set external to both scripts.

An alternative approach would be to add a `-v` option to `smoke-em.sh` script, where it simply forwards the supplied option value to `smoke-test.sh`.

I don't have a strong opinion on which approach is better, but note that currently (all?) other options are set via explicit command-line arguments and that the command-line arguments are documented through the `-h`/`-?` help. ... so perhaps I'm leaning more towards preferring command-line argument.

Reply:
Yup, agreed..  Will resolve this.